articles/aks/supported-kubernetes-versions.md

```diff
@@ -128,6 +130,38 @@ New Supported Version List
 1.17.*9*, 1.17.*8*, 1.16.*11*, 1.16.*10*
 ```
 
+## Platform Support Policy
+For one GA minor version after the Kubernetes support policy, AKS provides platform support. The platform support window of Kubernetes versions on AKS is known as "N-3".
+
+During platform support, customers running N-3 versions will only receive support from Microsoft for AKS platform related issues. Any issues related to the upstream Kubernetes community will not be supported. 
```

Comment:
AKS/Azure

related to Kubernetes functionality and components.

We also need to explain why in the docs. Like the blog it's crucial we explain in most places that
due to the nature of AKS relying on vanilla kurbernetes, which is an Open Source project that supports a sliding window of 3 versions (links). AKS can only guarantee full support (link to full support definition) while those versions are being serviced upstream. Since there's no more patches being produced upstream AKS can either not patch those and leave those versions unpatched or fork.

Reply:
a version of this should be in multiple places